config/settings.yml

```diff
@@ -3,9 +3,51 @@
   :ems_hawkular:
     :event_handling:
       :event_groups:
+
+:ems_refresh:
+  :hawkular:
+    :refresh_interval: 15.minutes
+
+:event_handling:
```

Comment:
this needs to be nested under `ems->ems_hawkular` because merging settings will not merge arrays. It will merge hashes though.